src/macros/for_others.rs

```diff
@@ -109,6 +111,19 @@ macro_rules! implement_for_others {
             }
         })*
     };
+
+    // [`Pow`] trait
+    ($bridge_type:ident, $type:ident, Pow for $($source_type:ident)*) => {
+        $(impl Pow<$source_type> for &$type {
+            type Output = $type;
+            paste::paste! {
+                #[doc = "Documentation can be found at [`" $type "::pow`]. Implicitly converts [`" $source_type "`] into [`" $bridge_type "`]."]
```

Comment:
Link in documnetation does not work since it is not implementeed for $type but rather for &$type. While you fix it, please also fix the documentation for Mul Scalar above, since it refers to `set_entry`. In order to fix it I propose a change to the trait implementation above.

Reply:
It could be possible that we have multiple implementations for scalar multiplication via Phil's trait and Marcel's implementation of `mul_scalar` for MatZ. This topic should probably be discussed in the group and the macros entirely refactored after the next sprint.